schema.graphql

```diff
@@ -19,6 +19,9 @@ type Query {
         recommendationCount: Int = 10
     ): Slate
 
+    # TODO: Is there a better name for `RecommendationResult`?
+    getCorpusSlate(slateId: String!): CorpusSlate
```

Comment:
We build new types so that we can deliver `CorpusItemRecommendation` which keys its content on `CorpusItem` from the curation tools.